packages/db/src/runtime/db-client.ts

```diff
@@ -1,58 +1,66 @@
 import type { InStatement } from '@libsql/client';
-import { createClient } from '@libsql/client';
-import { getTableName } from 'drizzle-orm';
+import { LibsqlError, createClient } from '@libsql/client';
 import type { LibSQLDatabase } from 'drizzle-orm/libsql';
 import { drizzle as drizzleLibsql } from 'drizzle-orm/libsql';
-import { type SQLiteTable } from 'drizzle-orm/sqlite-core';
 import { drizzle as drizzleProxy } from 'drizzle-orm/sqlite-proxy';
 import { z } from 'zod';
-import { type DBTables } from '../core/types.js';
+import type { SqliteDB } from './index.js';
+import { SEED_ERROR } from '../core/errors.js';
+import type { DBTables } from '../core/types.js';
+import { recreateTables } from './queries.js';
 
 const isWebContainer = !!process.versions?.webcontainer;
 
-interface LocalDatabaseClient extends LibSQLDatabase, Disposable {}
-
-export async function createLocalDatabaseClient({
-	tables,
-	dbUrl,
-	seeding,
-}: {
-	dbUrl: string;
-	tables: DBTables;
-	seeding: boolean;
-}): Promise<LocalDatabaseClient> {
+export function createLocalDatabaseClient({ dbUrl }: { dbUrl: string }): LibSQLDatabase {
 	const url = isWebContainer ? 'file:content.db' : dbUrl;
 	const client = createClient({ url });
-	const db = Object.assign(drizzleLibsql(client), {
-		[Symbol.dispose || Symbol.for('Symbol.dispose')]() {
-			client.close();
-		},
-	});
-
-	if (seeding) return db;
+	const db = drizzleLibsql(client);
 
-	const { insert: drizzleInsert, update: drizzleUpdate, delete: drizzleDelete } = db;
-	return Object.assign(db, {
-		insert(Table: SQLiteTable) {
-			checkIfModificationIsAllowed(tables, Table);
-			return drizzleInsert.call(this, Table);
-		},
-		update(Table: SQLiteTable) {
-			checkIfModificationIsAllowed(tables, Table);
-			return drizzleUpdate.call(this, Table);
-		},
-		delete(Table: SQLiteTable) {
-			checkIfModificationIsAllowed(tables, Table);
-			return drizzleDelete.call(this, Table);
-		},
-	});
+	return db;
 }
 
-function checkIfModificationIsAllowed(tables: DBTables, Table: SQLiteTable) {
-	const tableName = getTableName(Table);
-	const collection = tables[tableName];
-	if (!collection.writable) {
-		throw new Error(`The [${tableName}] collection is read-only.`);
+/**
+ * Sorted by precedence.
+ * Ex. If both "seed.dev.ts" and "seed.ts" are present,
+ * "seed.dev.ts" will be used.
+ */
+export const SEED_DEV_FILE_NAMES_SORTED = [
```

Comment:
do we need this feature for launch? I'd rather pick one of these and document it, vs. the complexity of the user reading docs and thinking "wait, is seed.development.ts different from seed.ts?"

Reply:
Was planning on a `seed.prod.ts` that runs with a `pnpm astro db seed` command. This would be a shortcut to running `pnpm astro db execute db/seed.whatever.ts`. If you don't think this is worth pursuing, I will remove!

Reply:
Update: decided `astro db seed` isn't very compelling. Just seed.ts, and `astro db execute` for any other file. Easy to doc